.kitchen.cloud.yml

```diff
@@ -74,6 +77,8 @@ suites:
     run_list:
       - role[openstack_ops_database]
       - role[openstack_controller]
+      - recipe[openstack-integration-test::setup]
+      - recipe[scl]
```

Comment:
Yes please. It only belongs in the metadata if the cookbook uses it directly. If TK uses it, just the berksfile.